util/env_windows_test.cc

```diff
@@ -1,65 +1,131 @@
-// Copyright (c) 2018 The LevelDB Authors. All rights reserved.
```

Comment:
I think something went wrong here, it's rewriting the entire file

Reply:
Thx, will check.